src/nwb_conversion_tools/datainterfaces/ecephys/intan/intandatainterface.py

```diff
@@ -79,6 +79,21 @@ def __init__(
         spikeextractors_backend: Optional[bool] = False,
         verbose: Optional[bool] = True,
     ):
+        """Load and prepare raw data and corresponding metadata from the Intan format (.rhd or .rhs files).
+
+
+        Parameters
+        ----------
+        file_path : FilePathType
+            Path to either a rhd or a rhs file
+        stream_id : Optional[str], optional
+            The stream of the data for spikeinterface, "0" by default.
+        spikeextractors_backend : Optional[bool], optional
+            False by default. When True the interface uses the old extractor from the spikextractors library instead
+            of a new spikeinterface object.
+        verbose : Optional[bool], optional
```

Comment:
```suggestion
        spikeextractors_backend: bool = False,
        verbose: bool = True,
    ):
        """Load and prepare raw data and corresponding metadata from the Intan format (.rhd or .rhs files).


        Parameters
        ----------
        file_path : FilePathType
            Path to either a rhd or a rhs file
        stream_id : str, optional
            The stream of the data for spikeinterface, "0" by default.
        spikeextractors_backend : bool
            False by default. When True the interface uses the old extractor from the spikextractors library instead
            of a new spikeinterface object.
        verbose : bool
```

Reply:
`Optional` is used from accepting the `None` value, so it is not appropriate here: https://docs.python.org/3/library/typing.html#typing.Optional

Reply:
Thanks, I added this somewhere else where it should not be. I will fix.